content/docs/faq-styling.md

```diff
@@ -32,20 +32,20 @@ render() {
 >
 >If you often find yourself writing code like this, [classnames](https://www.npmjs.com/package/classnames#usage-with-reactjs) package can simplify it.
 
-### Can I use inline styles? {#can-i-use-inline-styles}
+### Dapatkah saya menggunakan style di barisan? {#can-i-use-inline-styles}
 
-Yes, see the docs on styling [here](/docs/dom-elements.html#style).
+Ya, lihat dokumen pada styling [disini](/docs/dom-elements.html#style).
 
-### Are inline styles bad? {#are-inline-styles-bad}
+### Apakah style di barisan buruk? {#are-inline-styles-bad}
 
-CSS classes are generally better for performance than inline styles.
+Kelas CSS umumnya lebih baik untuk performa daripada style di barisan.
 
-### What is CSS-in-JS? {#what-is-css-in-js}
+### Apa yang dimaksud dengan CSS-dalam-JS? {#what-is-css-in-js}
```

Comment:
Shouldn't it be better to leave `CSS-in-JS` as it is rather than translating it?